sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <v-card>
+    <v-container>
+      <body-header title="History of Bypassed Restrictions" class="m-1">
+        <template #actions>
+          <check-permission-role :role="Role.StudentAddRestriction">
+            <template #="{ notAllowed }">
+              <v-btn
+                @click="addBypassedRestriction"
+                class="float-right"
+                color="primary"
+                data-cy="addRestrictionButton"
+                prepend-icon="fa:fa fa-plus-circle"
+                :disabled="notAllowed"
+                >Bypass A Restriction</v-btn
+              >
+            </template>
+          </check-permission-role>
+        </template>
+      </body-header>
+      <content-group class="mt-4">
+        <toggle-content
+          :toggled="!bypassedRestrictions.length"
+          message="No bypassed restrictions found."
+        >
+          <DataTable
+            :value="bypassedRestrictions"
+            :paginator="true"
+            :rows="DEFAULT_PAGE_LIMIT"
+            :rowsPerPageOptions="PAGINATION_LIST"
+            :totalRecords="bypassedRestrictions.length"
+          >
+            <template #empty>
+              <p class="text-center font-weight-bold">No records found.</p>
+            </template>
+            <Column
+              field="restrictionType"
+              header="Restriction Type"
+              :sortable="true"
+            ></Column>
+            <Column field="restrictionCode" header="Restriction Code"></Column>
+            <Column header="View Details">
```

Comment:
As per the technical AC, the column Restriction status using the status chip is missing. 
![image](https://github.com/user-attachments/assets/05b4318e-1679-4dad-b232-997404f24434)

I understand that image might be distracting us from the AC. We can see if we can do something better on the images with it's connection to ACs in future.

Reply:
I just took a look at the PR description image, even with the column name being "Restriction type" the content in the ticket seems to be from `restriction_category`.
![image](https://github.com/user-attachments/assets/a53b52c2-3795-4454-b64d-f6f3a114f81c)

Reply:
Updated.

Reply:
The comment from @andrewsignori-aot needs to be checked with business if they are expecting the restriction category as the restriction type `Federal | Provincial` is not very useful IMO.